src/components/UserDataGrid/UserColumns.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  DataGridProProps as DataGridProps,
+  GridColDef,
+  GridToolbarColumnsButton,
+  GridToolbarFilterButton,
+} from '@mui/x-data-grid-pro';
+import { RoleLabels, RoleList } from '~/api/schema.graphql';
+import {
+  booleanColumn,
+  getInitialVisibility,
+  multiEnumColumn,
+  QuickFilterButton,
+  QuickFilterResetButton,
+  QuickFilters,
+  textColumn,
+  Toolbar,
+  useFilterToggle,
+} from '../Grid';
+import { LinkColumn } from '../Grid/Columns/LinkColumn';
+import { UserDataGridRowFragment as User } from './userDataGridRow.graphql';
+
+export const UserColumns: Array<GridColDef<User>> = [
+  {
+    headerName: 'Full Name',
+    field: 'fullName',
+    ...textColumn(),
+    width: 350,
+    hideable: false,
+    valueGetter: (_, row) => row.fullName,
+    serverFilter: (value) => ({ name: value }),
+  },
+  LinkColumn({
+    field: 'User',
+    headerName: '',
+    valueGetter: (_, user) => user,
+    destination: (id) => `/users/${id}`,
+    width: 60,
+  }),
```

Comment:
Let's replace this column by just rendering the name column as a link.
https://github.com/SeedCompany/cord-field/blob/4d1926c29070926b0ea40aea3a5ec0d0ee8c8089/src/components/Grid/Columns/ProjectNameColumn.tsx#L27-L30